pkg/recordset/recordset.go

```diff
@@ -20,6 +20,28 @@ const (
 	installationTag = "giantswarm.io/installation"
 )
 
+var (
+	stackStatusCompleteNotDelete = []string{
+		cloudformation.StackStatusCreateComplete,
+		cloudformation.StackStatusRollbackComplete,
+		cloudformation.StackStatusUpdateComplete,
+		cloudformation.StackStatusUpdateRollbackComplete,
+	}
+	stackStatusCompleteNotDeleteAndFail = []string{
+		cloudformation.StackStatusCreateComplete,
+		cloudformation.StackStatusRollbackComplete,
+		cloudformation.StackStatusUpdateComplete,
+		cloudformation.StackStatusUpdateRollbackComplete,
+		cloudformation.StackStatusCreateFailed,
+		cloudformation.StackStatusRollbackFailed,
+		cloudformation.StackStatusDeleteFailed,
+		cloudformation.StackStatusUpdateRollbackFailed,
```

Comment:
It would be way easier to read when these list items would be ordered alphabetically.

Reply:
status alphabetically sorted

Reply:
![thumbs_up](https://user-images.githubusercontent.com/552769/47994905-07110c80-e0f4-11e8-95a5-5f6f818250fa.jpg)